### ISSUE TITLE
feat: Make Modal prompts accessible by screen readers

### ISSUE THREAD

**Issue by datawaslost** [ionic-team/capacitor#2310]
# Feature Request

## Describe the Feature Request
Currently, using Modal.prompt() triggers an Accessibility warning when submitting to the Android Google Play Store, because the android:id​/custom​/EditText[0] item doesn't have a screen-readable label.

## Platform Support Requested

- [x] Android
- [ ] iOS
- [ ] Electron
- [ ] Web

## Describe Preferred Solution
Add an inputLabel property to Modal PromptOptions, and implement this label in the Modal prompt.

## Related Code
```
let promptRet = await Modals.prompt({
	title: 'Title',
	message: "Message"
});
```

## Additional Context

From Google's Pre-Launch Report:

"This item may not have a label readable by screen readers."
https://support.google.com/accessibility/android/answer/7158690

Tested on:
Moto G4 Play
Android 6.0
English (United States)
720 × 1280
320

**Comment by jcesarmobile** [comment]
does the warning still appear if you use inputPlaceholder?

But just noticed the inputPlaceholder is a text and not a real placeholder, I've reported it in a separate issue.

https://github.com/ionic-team/capacitor/issues/2339

**Comment by jcesarmobile** [comment]
Closing since there was no response, I think using inputPlaceholder should be enough to get rid of the warning.
But as I said, it should be a placeholder and not text, but I reported that already and will use that issue to fix it.
